pkg/collector/python/util.go

```diff
@@ -79,3 +81,15 @@ func GetSubprocessOutput(argv **C.char, cStdout **C.char, cStderr **C.char, cRet
 	*cStderr = TrackedCString(string(outputErr))
 	*cRetCode = C.int(retCode)
 }
+
+var obfuscator = obfuscate.NewObfuscator(nil)
```

Comment:
Is there a way not to have this in global ? Is it costly to instantiate every time ? Does the obfucator keep an internal state ?

Reply:
It doesn't keep state. It's pretty lightweight, mostly [this config struct](https://github.com/DataDog/datadog-agent/blob/983b9792704458a4a6c7aa820a4a7385ede1765a/pkg/trace/obfuscate/obfuscate.go#L26-L25), which with the default settings is ~ 25 bytes. 

It'll need to be re-allocated on the heap each time the function runs if we recreate the obfuscator. Probably not a big deal given that this will be called only from python.

Is there a reason to avoid keeping it as a global?

Reply:
We try to allow package to be tear down or created multiple times if needed (for example you can instantiate multiple forwarder or aggregator if needed). The design of the agent was to have flexible package in `pkg` put together by binaries in `cmd`.

It it doesn't keep state I think this is fine